### PR TITLE
Update New-AdfsAccessControlPolicy.md

### PR DESCRIPTION
**Pull request by msbemba**
added instructions to get the XML file

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2784

File: docset/winserver2016-ps/adfs/New-AdfsAccessControlPolicy.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ New-AdfsAccessControlPolicy -Name <String> [-SourceName <String>] [-Identifier <
 ## DESCRIPTION
 The **New-AdfsAccessControlPolicy** cmdlet creates an Active Directory Federation Services (AD FS) access control policy from a policy metadata file.
 
+## How to create a Federation metadata file?
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/adfs/New-AdfsAccessControlPolicy.md:27]:
```suggestion
## How to create a Federation metadata file
```

File: docset/winserver2016-ps/adfs/New-AdfsAccessControlPolicy.md [hunk 2]
```diff
@@ -24,6 +24,10 @@ New-AdfsAccessControlPolicy -Name <String> [-SourceName <String>] [-Identifier <
...
+
+The federation metadata document is a XML file that is available for [download](https://adfshelp.microsoft.com/MetadataExplorer/GetFederationMetadata).Enter in your federation service name and click the 'Get federation metadata' button to retrieve your document.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/adfs/New-AdfsAccessControlPolicy.md:29]: ****
```suggestion
The federation metadata document is an XML file that is available for [download](https://adfshelp.microsoft.com/MetadataExplorer/GetFederationMetadata). Enter your federation service name and select the **Get federation metadata** button to retrieve your document.
```

File: docset/winserver2019-ps/adfs/New-AdfsAccessControlPolicy.md [hunk 3]
```diff
@@ -24,6 +24,11 @@ New-AdfsAccessControlPolicy -Name <String> [-SourceName <String>] [-Identifier <
 ## DESCRIPTION
 The **New-AdfsAccessControlPolicy** cmdlet creates an Active Directory Federation Services (AD FS) access control policy from a policy metadata file.
 
+## How to create a Federation metadata file?
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/adfs/New-AdfsAccessControlPolicy.md:27]:
```suggestion
## How to create a Federation metadata file?
```
```suggestion
## How to create a federation metadata file
```

File: docset/winserver2019-ps/adfs/New-AdfsAccessControlPolicy.md [hunk 4]
```diff
@@ -24,6 +24,11 @@ New-AdfsAccessControlPolicy -Name <String> [-SourceName <String>] [-Identifier <
...
+
+The federation metadata document is a XML file that is available for [download](https://adfshelp.microsoft.com/MetadataExplorer/GetFederationMetadata).Enter in your federation service name and click the 'Get federation metadata' button to retrieve your document.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/adfs/New-AdfsAccessControlPolicy.md:29]:
```suggestion
The federation metadata document is an XML file that is available for [download](https://adfshelp.microsoft.com/MetadataExplorer/GetFederationMetadata). Enter your federation service name and select the **Get federation metadata** button to retrieve your document.
```

File: docset/winserver2022-ps/adfs/New-AdfsAccessControlPolicy.md [hunk 5]
```diff
@@ -24,6 +24,10 @@ New-AdfsAccessControlPolicy -Name <String> [-SourceName <String>] [-Identifier <
 ## DESCRIPTION
 The **New-AdfsAccessControlPolicy** cmdlet creates an Active Directory Federation Services (AD FS) access control policy from a policy metadata file.
 
+## How to create a Federation metadata file?
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/adfs/New-AdfsAccessControlPolicy.md:27]:
```suggestion
## How to create a federation metadata file
```

File: docset/winserver2022-ps/adfs/New-AdfsAccessControlPolicy.md [hunk 6]
```diff
@@ -24,6 +24,10 @@ New-AdfsAccessControlPolicy -Name <String> [-SourceName <String>] [-Identifier <
...
+
+The federation metadata document is a XML file that is available for [download](https://adfshelp.microsoft.com/MetadataExplorer/GetFederationMetadata).Enter in your federation service name and click the 'Get federation metadata' button to retrieve your document.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/adfs/New-AdfsAccessControlPolicy.md:29]:
```suggestion
The federation metadata document is an XML file that is available for [download](https://adfshelp.microsoft.com/MetadataExplorer/GetFederationMetadata). Enter your federation service name and select the **Get federation metadata** button to retrieve your document.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by get-itips** [comment]
@msbemba Please review build errors, the header is not expected, maybe replace with a note.

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]
@msbemba, as @get-itips suggested, please move it to a note or just another sentence in the description section.

Thanks!

**Comment by msbemba** [comment]
> @msbemba, as @get-itips suggested, please move it to a note or just another sentence in the description section.
> 
> Thanks!

Thank you. the changes are completed.

File: docset/winserver2016-ps/adfs/New-AdfsAccessControlPolicy.md [hunk 7]
```diff
@@ -24,6 +24,10 @@ New-AdfsAccessControlPolicy -Name <String> [-SourceName <String>] [-Identifier <
 ## DESCRIPTION
 The **New-AdfsAccessControlPolicy** cmdlet creates an Active Directory Federation Services (AD FS) access control policy from a policy metadata file.
 
+**How to create a federation metadata file?**
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/adfs/New-AdfsAccessControlPolicy.md:27]:
```suggestion
**How to create a federation metadata file**
```

File: docset/winserver2019-ps/adfs/New-AdfsAccessControlPolicy.md [hunk 8]
```diff
@@ -24,6 +24,11 @@ New-AdfsAccessControlPolicy -Name <String> [-SourceName <String>] [-Identifier <
 ## DESCRIPTION
 The **New-AdfsAccessControlPolicy** cmdlet creates an Active Directory Federation Services (AD FS) access control policy from a policy metadata file.
 
+**How to create a federation metadata file?**
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/adfs/New-AdfsAccessControlPolicy.md:27]:
```suggestion
**How to create a federation metadata file**
```

File: docset/winserver2022-ps/adfs/New-AdfsAccessControlPolicy.md [hunk 9]
```diff
@@ -24,6 +24,10 @@ New-AdfsAccessControlPolicy -Name <String> [-SourceName <String>] [-Identifier <
 ## DESCRIPTION
 The **New-AdfsAccessControlPolicy** cmdlet creates an Active Directory Federation Services (AD FS) access control policy from a policy metadata file.
 
+**How to create a federation metadata file?**
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/adfs/New-AdfsAccessControlPolicy.md:27]:
```suggestion
**How to create a federation metadata file**
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, please commit the changes to remove the question marks again. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2016-ps/adfs/New-AdfsAccessControlPolicy.md [hunk 10]
```diff
@@ -24,6 +24,10 @@ New-AdfsAccessControlPolicy -Name <String> [-SourceName <String>] [-Identifier <
 ## DESCRIPTION
 The **New-AdfsAccessControlPolicy** cmdlet creates an Active Directory Federation Services (AD FS) access control policy from a policy metadata file.
 
+**How to create a federation metadata file**
+
+The federation metadata document is an XML file that is available for [download](https://adfshelp.microsoft.com/MetadataExplorer/GetFederationMetadata). Enter your federation service name and select the **Get federation metadata** button to retrieve your document.
```
Comment (tiburd) [inline docset/winserver2016-ps/adfs/New-AdfsAccessControlPolicy.md:29]:
```suggestion
The federation metadata document is an XML file that is available for [download](https://adfshelp.microsoft.com/MetadataExplorer/GetFederationMetadata). To retrieve your document, enter your federation service name, and then select the **Get federation metadata** button.
```

File: docset/winserver2019-ps/adfs/New-AdfsAccessControlPolicy.md [hunk 11]
```diff
@@ -24,6 +24,11 @@ New-AdfsAccessControlPolicy -Name <String> [-SourceName <String>] [-Identifier <
 ## DESCRIPTION
 The **New-AdfsAccessControlPolicy** cmdlet creates an Active Directory Federation Services (AD FS) access control policy from a policy metadata file.
 
+**How to create a federation metadata file**
+
+The federation metadata document is an XML file that is available for [download](https://adfshelp.microsoft.com/MetadataExplorer/GetFederationMetadata). Enter your federation service name and select the **Get federation metadata** button to retrieve your document.
```
Comment (tiburd) [inline docset/winserver2019-ps/adfs/New-AdfsAccessControlPolicy.md:29]:
```suggestion
The federation metadata document is an XML file that is available for [download](https://adfshelp.microsoft.com/MetadataExplorer/GetFederationMetadata). To retrieve your document, enter your federation service name, and then select the **Get federation metadata** button.
```

File: docset/winserver2022-ps/adfs/New-AdfsAccessControlPolicy.md [hunk 12]
```diff
@@ -24,6 +24,10 @@ New-AdfsAccessControlPolicy -Name <String> [-SourceName <String>] [-Identifier <
 ## DESCRIPTION
 The **New-AdfsAccessControlPolicy** cmdlet creates an Active Directory Federation Services (AD FS) access control policy from a policy metadata file.
 
+**How to create a federation metadata file**
+
+The federation metadata document is an XML file that is available for [download](https://adfshelp.microsoft.com/MetadataExplorer/GetFederationMetadata). Enter your federation service name and select the **Get federation metadata** button to retrieve your document.
```
Comment (tiburd) [inline docset/winserver2022-ps/adfs/New-AdfsAccessControlPolicy.md:29]:
```suggestion
The federation metadata document is an XML file that is available for [download](https://adfshelp.microsoft.com/MetadataExplorer/GetFederationMetadata). To retrieve your document, enter your federation service name, and then select the **Get federation metadata** button.
```

**Pull request merged**